src/frontend/src/lib/components/ui/Button.svelte

```diff
@@ -13,15 +14,20 @@
 </script>
 
 <button
-	class={`${colorStyle} flex flex-1 text-center ${styleClass}`}
+	class={`${colorStyle} flex flex-1 text-center transition duration-500 ease-in-out ${styleClass}`}
 	class:padding-sm={paddingSmall}
 	class:w-full={fullWidth}
 	class:link
 	{type}
 	{disabled}
```

Comment:
btw. can a button be set as disabled and loading at the same time?

Reply:
i wanted to do it at usage level: you mean to set the logic here right? If `loading` then `disabled` for sure?

Reply:
I just meant:

- Should a button be disabled when it is set as "loading"?

- Can a button be set as "disabled" and "loading" at the same time?

Reply:
ah! understood now

- you are right, i changed it so that in `<button>` there is  `disabled={disabled || loading}`; I simulated and tested and the `loading` visual state override the `disabled` settings, but the button is not clickable (as expected from `disabled`)

- as per comment above, I don't see why not, meaning that (I tested):
  1. `disabled` and not `loading` --> just `disabled`
  2. not `disabled` and `loading` --> `disabled` and `loading` (visually just `loading`)
  3. `disabled` and `loading` --> `disabled` and `loading` (visually just `loading`)
  
WDYT?

Reply:
done